modern-business/style-editor.css

```diff
@@ -981,9 +981,15 @@ body .wp-block[data-align="full"] {
   width: 100%;
 }
 
+/** === Editor background color === */
+.editor-styles-wrapper {
+  background: #fff;
```

Comment:
This is pretty odd, inspecting in FF/Chrome this property isn't even set at all. (Not inherited from body either).

<img width="651" alt="Screen Shot 2019-08-07 at 12 02 35 PM" src="https://user-images.githubusercontent.com/1270189/62650280-5b5b4e00-b90b-11e9-839b-3d54ccc6e86a.png">

@noahtallen have you tried a different selector, or doubling up there with an additional class?

One thing I noted was a style tag at the bottom page for related `editor-styles-wrapper` rules.

Reply:
Oh my bad, wrong prop name:

<img width="696" alt="Screen Shot 2019-08-07 at 12 06 10 PM" src="https://user-images.githubusercontent.com/1270189/62650479-d3297880-b90b-11e9-9c38-f76e0d00a8ec.png">

Reply:
<img width="546" alt="Screen Shot 2019-08-07 at 12 12 21 PM" src="https://user-images.githubusercontent.com/1270189/62650908-b04b9400-b90c-11e9-92fc-1383eef69d72.png">

Here's why the rule isn't applying. I forget which is which, but we're likely pulling in the sass as a mixin or extends.

`.editor-styles-wrapper .editor-styles-wrapper` won't apply.

Reply:
Oh that's really odd. I kinda thought that would show up in the output CSS. Good catch!

Reply:
Ah as Noah noted, the extra class is coming from default theme support behavior https://github.com/WordPress/gutenberg/blob/0ad0414e7a469254ef1ad105fdcf14248a986686/docs/designers-developers/developers/themes/theme-support.md